snuba/clickhouse/query.py

```diff
@@ -19,7 +17,7 @@ class ClickhouseQuery(ABC):
     """
 
     @abstractmethod
-    def format_sql(self) -> str:
+    def format_sql(self, format: Optional[str] = None) -> str:
```

Comment:
There's a bunch of stuff in this file that Black wants to reformat, I skipped checks to avoid noise in this patch.